sds/src/views/home/Home.vue

```diff
@@ -224,8 +224,8 @@
 
         <v-row>
           <v-col cols="12" md="6">
-            <div class="txt-box-side">
-              <LottieIcon icon="wired-gradient-436-love-care.json" />
+            <div class="txt-box-side" style="padding-top: 8px;">
+              <img src="../../assets/images/content/SDSAccessIconsAsset18.svg" height="36px">
               <div class="b1-heading">
```

Comment:
@rdell is this an intended change?

Reply:
@bufordeeds I believe this is that icon that is actually 3 or 4 icons together. Marc wanted this to replace the icon you used.

Reply:
Sounds good!